### ISSUE TITLE
How to use the KeychainHelpers module?

### ISSUE THREAD

**Issue by gbrhaz** [calabash/calabash-ios#1310]
I want to use `Calabash::Cucumber::KeychainHelpers` but I'm getting some undefined method errors.

If I do something like:

```
class Keychain
  include Calabash::Cucumber:KeychainHelpers
end
```

and then do `Keychain.new.keychain_clear` I get an error that says something along the lines of:

`undefined method http on Keychain`

I see that `http` is a method that is called in the `_keychain_post` method, and is in the Core module. So this maybe more of a Ruby question... but what's the solution for this? 

Essentially, my goal is to clear the keychain on device as a way of resetting it (it seems to be the best way for me right now)

**Comment by jmoody** [comment]
* [keychain.feature](https://github.com/calabash/ios-smoke-test-app/blob/master/CalSmokeApp/features/keychain.feature)
* [keychain.rb](https://github.com/calabash/ios-smoke-test-app/blob/master/CalSmokeApp/features/step_definitions/keychain_steps.rb)

### TL;DR

Just call `keychain_clear` in your Before hook after launching.

**Comment by gbrhaz** [comment]
Yep, should've closed this before as I found the solution. It was including `::Operations` and calling it after the launcher had been created that did it for me.
